subtests/docker_cli/systemd/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM centos:latest
+FROM rhel:latest
```

Comment:
Well, I guess it has to be something like {from_img} since it is edited during test run